contrib/gofiber/fiber.v2/option.go

```diff
@@ -25,7 +26,7 @@ type Option func(*config)
 
 func defaults(cfg *config) {
 	cfg.serviceName = "fiber"
-	cfg.isStatusError = isServerError
+	cfg.isStatusError = globalconfig.IsHTTPServerError(cfg.statusCode)
```

Comment:
If this is correct, I can make the changes for the other integrations using this format as well to change the function signature for `WithStatusCheck` and adding a new config field for `statusCode`.